web/src/pages/Cases/CaseDetails/Evidence/index.tsx

```
@@ -93,12 +95,30 @@ const Evidence: React.FC = () => {
     latestEvidence.scrollIntoView({ behavior: "smooth" });
   }, [ref]);
 
+  const flattenedSpamEvidences = useMemo(
+    () =>
+      spamEvidences?.courtv2EvidenceSpamsByDeployment.reduce<string[]>((acc, current) => {
+        if (current.dispute === id) {
+          return [...acc, current.disputeEvidenceIndex];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)